### PR TITLE
working progress

### PR DESCRIPTION
**Pull request by carolinalp**
Hola amigos see if you like

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
 # hello-world
 This is my first repository and I created it to learn
 This what I changed when Isabella arrived from school
+Now we are trying to review what I have learnt
```
Comment (beatrizleon) [inline README.md]: I think it will be better to replace 
are trying to review 
with "reviewing" :)

File: README.md [hunk 2]
```diff
@@ -1,3 +1,4 @@
 # hello-world
 This is my first repository and I created it to learn
```
Comment (beatrizleon) [inline README.md:2]: Cambia esto tambien

**Review by beatrizleon — CHANGES_REQUESTED** [review]
Please change it according to my comments

File: README.md [hunk 3]
```diff
@@ -1,3 +1,4 @@
...
 This what I changed when Isabella arrived from school
```
Comment (beatrizleon) [inline README.md]:
```suggestion
This what I changed when Isabella arrived from evergreen school
```

**Comment by carolinalp** [comment]
ready! waiting for the merge

**Review by beatrizleon — APPROVED** [review]

**Pull request merged**